examples/src/main/python/streaming/mqtt_wordcount.py

```diff
@@ -0,0 +1,58 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+ A sample wordcount with MqttStream stream
+ Usage: mqtt_wordcount.py <broker url> <topic>
+ To work with Mqtt, Mqtt Message broker/server required.
+ Mosquitto (http://mosquitto.org/) is an open source Mqtt Broker
+ In ubuntu mosquitto can be installed using the command  `$ sudo apt-get install mosquitto`
+ Run Mqtt publisher as
+  `$ bin/run-example \
```

Comment:
bin/run-example is also inside spark, I misunderstood that, sorry.

never-mind.